File: 프로그래머스/lv1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.*;

class Solution {
    public int[] solution(int[] numbers) {
        ArrayList<Integer> arr = new ArrayList<>();
        int tmp = 0;
        for(int i = 0; i < numbers.length-1; i++){
            for(int j = i+1; j < numbers.length; j++){
                tmp = numbers[i] + numbers[j];
                if(!arr.contains(tmp)){
                    arr.add(tmp);
                }
            }
        }
        int[] answer = new int[arr.size()];
        for(int i = 0; i < answer.length; i++){
            answer[i] = arr.get(i);
        }
        Arrays.sort(answer);
        return answer;
    }
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
import itertools

def solution(numbers):
    answer = []
    idx = [i for i in range(len(numbers))]
    npr = itertools.permutations((idx), 2)
    for i, j in list(npr):
        answer.append(numbers[i]+numbers[j])
    answer = list(set(answer))
    answer.sort()
    return answer